### ISSUE TITLE
bug: Bug in version 1.0.0-beta.14 for filter ng-repeat

### ISSUE THREAD

**Issue by milindsaraswala** [driftyco/ionic#2974]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>all</span>  

<span ionic-description>Below is my plnkr 
http://plnkr.co/edit/f6LYS2aGrTXGkZ3vrIDD?p=preview

Bug simulation is on Search Page

Where in I have reference http://code.ionicframework.com/1.0.0-beta.14/js/ionic.bundle.js ionic bundle. When I reference above js file than my second directive search filter will stop working but at the same time, if I reference http://code.ionicframework.com/1.0.0-beta.1/js/ionic.bundle.js it works search filter in both directive.

So in short there is bug in latest version or I am missing something which I have to follow. Kindly somebody from ionic team please come forward and help us.</span>

<span is-issue-template></span>

**Comment by milindsaraswala** [comment]
I got reply from some tasseKATT on stackoverflow on this URL http://stackoverflow.com/questions/28103165/angularjs-search-filter-issue/28105438#28105438. He have given me plunker URL http://plnkr.co/edit/ZAM33j82gT4Y6hqJLqAl?p=preview which shows that I have to use angularjs 1.3.8 or later 

So Kindly can you help me how can I update ionic to use new angularjs or where can I put my angularjs file in file structure.

Kindly help me as I new so I need to know. Hope to get reply soon.

**Comment by stianjensen** [comment]
The file lib/ionic/js/ionic-bundle.js which is included from index.html by default, is a concatenation of ionic.js, angular.js, angular-animate.js, angular-sanitize.js, angular-ui-router.js and ionic-angular.js.

You could try and include these files in your index.html and then switch out the angular component for version 1.3.8 (You might need to update some of the other angular components to, for compatibility with the angular update). Or, you could concatenate them yourself, to an updated ionic.bundle.js.

All the files are inside the lib/ionic/js folder, and new files can be downloaded from https://code.angularjs.org/1.3.8/angular.js and the like

Now, this might not work properly with ionic, because ionic still uses angular 1.3.6 even in the nightly builds.

**Comment by milindsaraswala** [comment]
Thank you for your reply. I will try to follow your instruction.

**Comment by ajoslin** [comment]
This works in the nightly builds.
